[0m00:48:23.571858 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E16EDF5CD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E16F2A6F10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E16F2A7110>]}


============================== 00:48:23.577859 | 98fd11f1-f6ea-459e-8db2-09f01fb5b789 ==============================
[0m00:48:23.577859 [info ] [MainThread]: Running with dbt=1.7.4
[0m00:48:23.578858 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\Nico\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m00:48:32.553465 [debug] [MainThread]: Starter project path: C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\include\starter_project
[0m00:48:32.576200 [info ] [MainThread]: 
Your new dbt project "dbt_transformation" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m00:48:32.576200 [info ] [MainThread]: Setting up your profile.
[0m00:58:15.826747 [error] [MainThread]: Encountered an error:

[0m00:58:15.829744 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\cli\requires.py", line 90, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\cli\requires.py", line 75, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\cli\main.py", line 515, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\task\init.py", line 344, in run
    self.setup_profile(profile_name)
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\task\init.py", line 264, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\task\init.py", line 181, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\task\init.py", line 165, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\dbt\task\init.py", line 131, in generate_target_from_input
    target[key] = click.prompt(
                  ^^^^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\click\termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\venv\Lib\site-packages\click\termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m00:58:15.833745 [debug] [MainThread]: Command `dbt init` failed at 00:58:15.833745 after 592.27 seconds
[0m00:58:15.834744 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E16EEF4B10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E170B58590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E16EB0EF90>]}
[0m00:58:15.836745 [debug] [MainThread]: Flushing usage events
[0m01:44:51.667936 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2F325550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2F325710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2F2BD650>]}


============================== 01:44:51.673144 | 23e9f621-e1dc-45b3-b546-d99b966d9106 ==============================
[0m01:44:51.673144 [info ] [MainThread]: Running with dbt=1.7.4
[0m01:44:51.673144 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\Nico\\Documents\\GitHub\\big-star-de-pipeline\\dbt_transformation\\config', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m01:44:51.675153 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Nico\Documents\GitHub\big-star-de-pipeline\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m01:44:51.679150 [debug] [MainThread]: Command `dbt run` failed at 01:44:51.677151 after 0.08 seconds
[0m01:44:51.679150 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2EB2F250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2EC9DF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000012C2F325550>]}
[0m01:44:51.679150 [debug] [MainThread]: Flushing usage events
